tests/test_lammps.py

```diff
@@ -73,6 +74,12 @@ def test_write_potential(n_species, n_types, power):
     forces_lmp = test_structure.get_forces()
     stress_lmp = test_structure.get_stress()
 
+    # add back single_atom_energies to lammps energy
+    if sgp_model.gp_model.single_atom_energies is not None:
+        for spec in test_structure.numbers:
+            coded_spec = sgp_model.gp_model.species_map[spec]
+            energy_lmp += sgp_model.gp_model.single_atom_energies[coded_spec]
```

Comment:
We should implement single atom energies in our LAMMPS potential so that there's an exact match between the SGP and the potential file (this can be done in a future PR, but we should put it on the to-do list). We would just need to add a few lines to the header of the lammps potential file specifying the single-atom energies for each atom type, and update the PE compute.

Reply:
updated in Todos